designer/client/test/ProcessUtils-test.js

```diff
@@ -3,22 +3,20 @@ import { reject } from "lodash";
 
 const unknown = { display: "Unknown", params: [], type: "Unknown", refClazzName: "java.lang.Object" };
 
-//nodeId, process, processDefinition, fieldName, processCategory  ==> (processDefinition, processCategory, process) => (nodeId, parameterDefinition)
+//nodeId, process, processDefinition, fieldName  ==> (processDefinition, process) => (nodeId, parameterDefinition)
```

Comment:
I'm not sure what this comment represents, but looks like you do. Can you add one line of explanation for future generations? :)

Reply:
I don't as well :) I just know that processCategory doesn't exist here anymore. Let's remove this comment